### DISCUSSION TITLE
Icon Request:     Playground icon

### DISCUSSION THREAD

**Discussion by faithdong** [FortAwesome/Font-Awesome#21209] [Icon Requests]
Please help add a chart about the style of the playground。
Thanks

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Any example?

Please take another look at our [Contributing guidelines](https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#requesting-new-icons).

**Comment by KjellDaSensei** [comment]
I'm also looking for a playground icon.

Perhaps something like these examples:

![bilde](https://user-images.githubusercontent.com/10048176/44261317-91458500-a217-11e8-8eb8-8e88e8a00596.png)

![bilde](https://user-images.githubusercontent.com/10048176/44261366-ba661580-a217-11e8-8f41-f23dffb68dd7.png)

That last one looks a bit too much like an adult on a children's slide, but the general idea is still the same.

**Comment by jeisel-northglenn** [comment]
I work for a city government - and a playground icon would be SUPER helpful.

**Comment by beckyyukon** [comment]
Also looking for a playground Icon. Thanks!

**Comment by leanda** [comment]
Also looking for this, slide, roundabout, swings, seesaw etc. would be a useful addition.

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
[🛝 U+1F6DD](https://emojipedia.org/emoji/%F0%9F%9B%9D/) Playground Slide

**Comment by nagarou-fr** [comment]
Also looking for a playground Icon. Thanks!

**Comment by TAR5** [comment]
+1
I also need this for a map.

**Comment by SerenityFire** [comment]
+1
I also need this for a client.
Would be super nice Thanks

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
